README.md

```diff
@@ -70,7 +71,7 @@ now
 ```
 
 
-## Using external components
+## Using external components (webpack only)
```

Comment:
Someone could read this and think you can't use external components with Rollup — how about this instead?

```suggestion
## Using external components with webpack
```